cmd/ocm-backplane/login/login.go

```diff
@@ -324,7 +334,15 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 
 	targetContext.AuthInfo = targetUserNickName
 	targetContext.Cluster = clusterName
-	targetContext.Namespace = "default"
+
+	logger.Debugln("Validating argument passed as namespace")
```

Comment:
Shouldn't this only be logged if `args.defaultNamespace!="default"`?

Reply:
> Shouldn't this only be logged if `args.defaultNamespace!="default"`?

Please review https://github.com/openshift/backplane-cli/pull/374/commits/bce3a48d8838f38d50a2539a948b9f13e9d70cdd .